homeassistant/components/easee/config_flow.py

```diff
@@ -0,0 +1,131 @@
+"""Config flow to configure Easee component."""
+import logging
+from typing import List, Optional
+
+from easee import AuthorizationFailedException, Easee, Site
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_MONITORED_CONDITIONS, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import callback
+from homeassistant.helpers import aiohttp_client, config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+from .const import (
+    CONF_MONITORED_SITES,
+    CUSTOM_UNITS,
+    CUSTOM_UNITS_OPTIONS,
+    DOMAIN,
+    EASEE_ENTITIES,
+    MEASURED_CONSUMPTION_DAYS,
+    MEASURED_CONSUMPTION_OPTIONS,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class EaseeConfigFlow(config_entries.ConfigFlow):
+    """Easee config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    @staticmethod
+    @callback
+    def async_get_options_flow(config_entry):
+        """Get the options flow for this handler."""
+        return OptionsFlowHandler(config_entry)
+
+    async def async_step_user(self, user_input: Optional[ConfigType] = None):
+        """Handle a flow start."""
+        # Supporting a single account.
+        entries = self.hass.config_entries.async_entries(DOMAIN)
+        if entries:
+            return self.async_abort(reason="already_setup")
+
+        errors = {}
+
+        if user_input is not None:
+            username = user_input[CONF_USERNAME]
+            password = user_input[CONF_PASSWORD]
+
+            try:
+                client_session = aiohttp_client.async_get_clientsession(self.hass)
+                easee = Easee(username, password, client_session)
+                # Check that login is possible
+                await easee.connect()
```

Comment:
Suggestion: when feasible, define and use a unique_id as written here: https://developers.home-assistant.io/docs/config_entries_config_flow_handler
Very useful if there will be more than one integration .